xgettext.js

```diff
@@ -26,21 +26,29 @@ XGettext.defaultOptions = {
 	 * number to return value in that argument position on a 1-based index.
 	 *
 	 * @type {Object}
- 	 * @see https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/Parser_API
+	 * @see https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/Parser_API
```

Comment:
Should we change this link to either the [Babel AST Spec](https://github.com/babel/babylon/blob/master/ast/spec.md) and/or the [ESTree Spec](https://github.com/estree/estree)?

From the Babylon docs: https://github.com/babel/babylon#output